src/ja/worker/main.py

```diff
@@ -43,7 +44,8 @@ def run(self) -> None:
         Registers this worker on the central server and starts the  main loop of the command handler.
         """
         register_response = self._server_proxy.register_self(
-            uid=self._config.uid, work_machine_resources=WorkMachineResources(self._config.resources))
+            uid=self._config.uid, work_machine_resources=WorkMachineResources(self._config.resources),
+            ssh_config=self._config.ssh_config)
```

Comment:
This config is wrong, and does not belong to this PR. I have assigned myself for the task and I'll submit a PR where this is fixed, so please remote it from here.

Reply:
will do

Reply:
or, just cherry-pick my commit and I'll close my PR. I realize that both are dependent on each other.